### DISCUSSION TITLE
Scrypted Accessory Mode Question

### DISCUSSION THREAD

**Discussion by RodneyCapron** [koush/scrypted#266] [General]
Hello @koush,

If you pair your devices in HomeKit accessory mode, do you still need to pair the Bridge?

Thank you!

Rodney

**Comment by koush** [comment]
only if you have stuff like lights etc on the bridge too.

**Reply by RodneyCapron** [reply]
Thanks again, @koush!

Have a great rest of the weekend! :)

**Comment by cheme75** [comment]
wait... we can add stuff like the lights and plugs?  how? what plugins?  i got my wyze plugs and bulbs still in hoobs - if I can unload that, then I think this 10 yr old pentium nettbook might be a lot happier!
